cmd/user-cluster-controller-manager/main.go

```diff
@@ -59,25 +59,26 @@ import (
 )
 
 type controllerRunOptions struct {
-	metricsListenAddr  string
-	healthListenAddr   string
-	version            string
-	networks           networkFlags
-	namespace          string
-	clusterURL         string
-	openvpnServerPort  int
-	kasSecurePort      int
-	tunnelingAgentIP   flagopts.IPValue
-	overwriteRegistry  string
-	cloudProviderName  string
-	nodelabels         string
-	seedKubeconfig     string
-	ownerEmail         string
-	updateWindowStart  string
-	updateWindowLength string
-	dnsClusterIP       string
-	opaIntegration     bool
-	useSSHKeyAgent     bool
+	metricsListenAddr            string
+	healthListenAddr             string
+	version                      string
+	networks                     networkFlags
+	namespace                    string
+	clusterURL                   string
+	openvpnServerPort            int
+	kasSecurePort                int
+	tunnelingAgentIP             flagopts.IPValue
+	overwriteRegistry            string
+	cloudProviderName            string
+	nodelabels                   string
+	seedKubeconfig               string
+	ownerEmail                   string
+	updateWindowStart            string
+	updateWindowLength           string
+	dnsClusterIP                 string
+	opaIntegration               bool
+	opaIntegrationWebhookTimeout int
```

Comment:
Can we call this "OPA webhook timeout" everywhere? the dditional "integration" sounds weird. ^^